### PR TITLE
Bump Contact model version number

### PR DESCRIPTION
**Pull request by smn**
After #901 @hodgestar recommends bumping the version number and adding an empty migration.

### CODE REVIEW

**Comment by smn** [comment]
@hodgestar @jerith ready for review.

File: go/vumitools/contact/old_models.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+from datetime import datetime
+
+from vumi.persist.model import Model, Manager
+from vumi.persist.fields import (Unicode, ManyToMany, ForeignKey, Timestamp,
+                                 Dynamic)
+
+from go.vumitools.account.old_models import UserAccountVNone
```
Comment (hodgestar) [inline go/vumitools/contact/old_models.py]: We're on UserAccountV4 currently?

Comment (smn) [inline go/vumitools/contact/old_models.py]: Doh.

**Comment by hodgestar** [comment]
Looks good but don't we need a test for the migration?

**Comment by hodgestar** [comment]
We also need a test for the migration itself -- i.e. save a vnone contact, load it with v1.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**